lua/nvim-lsp-installer/servers/omnisharp/init.lua

```diff
@@ -7,55 +7,83 @@ local std = require "nvim-lsp-installer.core.managers.std"
 
-            -- We write to the omnisharp directory for backwards compatibility reasons
-            ctx.fs:mkdir "omnisharp"
-            ctx:chdir("omnisharp", function()
-                github.unzip_release_file({
-                    repo = "OmniSharp/omnisharp-roslyn",
-                    asset_file = coalesce(
-                        when(
-                            platform.is_mac,
-                            coalesce(
-                                when(platform.arch == "x64", "omnisharp-osx-x64-net6.0.zip"),
-                                when(platform.arch == "arm64", "omnisharp-osx-arm64-net6.0.zip")
-                            )
-                        ),
-                        when(
-                            platform.is_linux,
-                            coalesce(
-                                when(platform.arch == "x64", "omnisharp-linux-x64-net6.0.zip"),
-                                when(platform.arch == "arm64", "omnisharp-linux-arm64-net6.0.zip")
-                            )
-                        ),
-                        when(
-                            platform.is_win,
-                            coalesce(
-                                when(platform.arch == "x64", "omnisharp-win-x64-net6.0.zip"),
-                                when(platform.arch == "arm64", "omnisharp-win-arm64-net6.0.zip")
-                            )
-                        )
-                    ),
-                }).with_receipt()
-            end)
-        end,
-        default_options = {
-            cmd = {
-                "dotnet",
-                path.concat { root_dir, "omnisharp", "OmniSharp.dll" },
-                "--languageserver",
-                "--hostPID",
-                tostring(vim.fn.getpid()),
-            },
-        },
-    }
+      -- We write to the omnisharp directory for backwards compatibility reasons
+      ctx.fs:mkdir "omnisharp"
+      ctx:chdir("omnisharp", function()
+        github.unzip_release_file({
+          repo = "OmniSharp/omnisharp-roslyn",
+          asset_file = coalesce(
+            when(
+              platform.is_mac,
+              coalesce(
+                when(platform.arch == "x64", "omnisharp-osx-x64-net6.0.zip"),
+                when(platform.arch == "arm64", "omnisharp-osx-arm64-net6.0.zip")
+              )
+            ),
+            when(
+              platform.is_linux,
+              coalesce(
+                when(platform.arch == "x64", "omnisharp-linux-x64-net6.0.zip"),
+                when(platform.arch == "arm64", "omnisharp-linux-arm64-net6.0.zip")
+              )
+            ),
+            when(
+              platform.is_win,
+              coalesce(
+                when(platform.arch == "x64", "omnisharp-win-x64-net6.0.zip"),
+                when(platform.arch == "arm64", "omnisharp-win-arm64-net6.0.zip")
+              )
+            )
+          ),
```

Comment:
It's probably fine, but this will make the installer fail if it's unable to find an appropriate net6.0 distribution. We seem to cover pretty much all OS and arch combinations so I think it's fine!

Reply:
The credit of this part is not for me, it is just how it was handled already.